libs/natparse/src/main/java/org/amshove/natparse/natural/IAtnOperandNode.java

```diff
@@ -0,0 +1,6 @@
+package org.amshove.natparse.natural;
+
+public interface IAtnOperandNode extends IOperandNode
```

Comment:
To simplify the code, lets create an interface `IMathFunctionOperandNode` that extends `IOperandNode` and make every one of these interfaces implement `IMathFunctionOperandNode`

If all these functions only have one parameter, the interface could be 

```java
public interface IMathFunctionOperandNode extend IOperandNode
{
    IOperandNode parameter();
}
```

Then we can make all interfaces like `IAtnOperandNode`, `IAverOperandNode` etc. empty.

Reply:
If I miss some functions that take more than one parameter my approach won't work :-)

Reply:
> If I miss some functions that take more than one parameter my approach won't work :-)

I’m quite sure the only take one. :)